frontend/src/components/Nav.js

```diff
@@ -4,14 +4,43 @@ import Router from '../core/Router.js';
 export default class Nav extends Component {
   async template() {
     return `
-      <a href="/" class="nav__link" data-link>home</a>
-      <a href="/user/dummy1" class="nav__link" data-link>user-dummy</a>
-      <a href="/game" class="nav__link" data-link>game</a>
-      <a href="/rank" class="nav__link" data-link>ranks</a>
-      <a href="/test" class="nav__link" data-link>test</a>
-      <a href="/login" class="nav__link" data-link>login-test</a>
-      <input type="button" value="login" />
-    `;
+<link rel="stylesheet" type="text/css" href="../../public/assets/nav/nav.css" />
```

Comment:
css를 한 폴더 안에 모아두는 것이 좋을 것 같습니다. 이미 만든 css folder가 있으니 그곳에 두는 것이 좋아 보입니다. 그리고 nav 폴더는 없어도 되지 않나 싶네요.
```suggestion
<link rel="stylesheet" type="text/css" href="../../public/assets/css/nav.css" />
```

Reply:
좋아요.  다른 브랜치에서 작업하는중인데 css 에 파일을 모아놓고 쓰고 있으니, 구조 변경하셔도 지장은 없을 것 같아요. nav 폴더는 처음에 갈아 끼우려고 할 때 뭔가 귀찮아질 것 같아서 그렇게 해둔거였는데, 옮겨도 될것 같아요